### ISSUE TITLE
docker-compose up build fails

### ISSUE THREAD

**Issue by jucamohedano** [ragapp/ragapp#71]
**Describe the bug**
When running the deployment command the building process fails.

**To Reproduce**
1. Run `MODEL=qwen2:0.5b docker-compose up`
4. See error:
```
127.4 ./app/components/ui/chat/chat-message/markdown.tsx:20:21
127.4 Type error: This regular expression flag is only available when targeting 'es2018' or later.
127.4 
127.4   18 |   // Replace block-level LaTeX delimiters \[ \] with $$ $$
127.4   19 |   const blockProcessedContent = content.replace(
127.4 > 20 |     /\\\[(.*?)\\\]/gs,
127.4      |                     ^
127.4   21 |     (_, equation) => `$$${equation}$$`,
127.4   22 |   );
127.4   23 |   // Replace inline LaTeX delimiters \( \) with $ $
127.5  ELIFECYCLE  Command failed with exit code 1.
127.5 make: *** [Makefile:33: build-chat] Error 1
------
Dockerfile:18
```

**Desktop (please complete the following information):**
 - OS: ubuntu20.04
 - Browser: chrome
 - Version 126.0.6478.55 (Official Build) (64-bit)

**Comment by Rauljauregi** [comment]
I realized that this is what is behind this other opened issue #35 @marcusschiesser

**Comment by tvlc** [comment]
Issue is resolved in version 0.1.12 of create_llama (https://github.com/run-llama/create-llama/releases/tag/v0.1.12).

To update version, just change **Makefile**:
`export CREATE_LLAMA_VERSION=0.1.12`

Now buids fine. 
See create_lama release info how the issue was resolved.

**Comment by marcusschiesser** [comment]
also fixed in latest RAGapp release
